consumer.py

```diff
@@ -38,6 +38,7 @@
 from thoth.messaging import SolvedPackageMessage
 from thoth.messaging import UnresolvedPackageMessage
 from thoth.messaging import UnrevsolvedPackageMessage
+from thoth.messaging import UpdateProvidesSourceDistroMessage
```

Comment:
this seems to be missing in https://github.com/thoth-station/messaging/blob/b2952d2abfb811c1724bb95b38caf137e501ab66/thoth/messaging/__init__.py
would give us module not found issue.

Reply:
very nice catch @harshad16!! do you want to take care?

Reply:
You can take it. 
Would be a short change and a release of messaging.

Reply:
https://github.com/thoth-station/messaging/pull/284 sure!

Reply:
new release of messaging added